cmd/syncEnumerator.go

```diff
@@ -62,7 +63,7 @@ func (cca *cookedSyncCmdArgs) initEnumerator(ctx context.Context) (enumerator *s
 			if entityType == common.EEntityType.File() {
 				atomic.AddUint64(&cca.atomicSourceFilesScanned, 1)
 			}
-		}, nil, cca.s2sPreserveBlobTags, cca.logVerbosity.ToPipelineLogLevel(), cca.cpkOptions)
+		}, nil, cca.s2sPreserveBlobTags, cca.logVerbosity.ToPipelineLogLevel(), cca.cpkOptions, nil)
```

Comment:
this and the next one in this file will be used for sync mode enumeration. Even there we would want the enumeration errors to flow like we are doing for the copy mode.
Since the sync work flow is currently not sorted for us, we can leave it, but we need to be aware of this.
Add a TODO here

// TODO: Consider passing an errorChannel so that enumeration errors during sync can be conveyed to the caller.

@nitin-deamon